resources/js/components/PaginationLinks.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
Is any of this custom, or is it all the same as a template? If it's the same, or even mainly the same as a template, let's put a note in here of where it came from and which pieces are custom.